dasharo-security/secure-boot.robot

```diff
@@ -233,6 +233,52 @@ SBO008.001 Attempt to enroll the key in the incorrect format (firmware)
     Select File In File Explorer    DB.txt
     Read From Terminal Until    ERROR: Unsupported file type!
 
+SBO009.001 Attempt to enroll the key in the incorrect format (OS)
+    [Documentation]    This test verifies that it is impossible to load
+    ...    a certificate in the wrong file format from the OS.
+    Skip If    not ${SECURE_BOOT_SUPPORT}    SBO008.001 not supported
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    SBO008.001 not supported
+    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    SBO008.001 not supported
+    Power On
+    ${sb_menu}=    Enter Secure Boot Menu And Return Construction
+    ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
+    Erase All Secure Boot Keys    ${advanced_menu}
+    Save Changes And Reset    3    5
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    # remove existing directory that contains the keys
+    Execute Linux Command    rm -rf /usr/share/secureboot/keys
+    # make the directory structure
+    Execute Linux Command    mkdir /usr/share/secureboot/keys
+    Execute Linux Command    mkdir /usr/share/secureboot/keys/PK
+    Execute Linux Command    mkdir /usr/share/secureboot/keys/KEK
+    Execute Linux Command    mkdir /usr/share/secureboot/keys/db
+    Execute Linux Command    mkdir /usr/share/secureboot/keys/dbx
+    # generate and move incorrectly formatted db keys to the right place
+    Execute Linux Command
+    ...    openssl ecparam -genkey -name secp384r1 -out db.key && openssl req -new -x509 -key db.key -out db.pem -days 365 -subj "/CN=db"
```

Comment:
can we set output into `/usr/share/secureboot/keys/db` instead of moving them in next commands? this way we will shrink the test commands.

Reply:
@TomaszAIR I made [this keyword](https://github.com/Dasharo/open-source-firmware-validation/pull/150/commits/306a15f47225151f12b9c3b63a7ef026dd5549df#diff-b06ff90d8f021dfa2e18d217f3d4c1e6861d6f671893cf292b6718b80227921aR221-R229), which makes the code much cleaner.